com.unity.netcode.gameobjects/Runtime/Configuration/NetworkPrefabs.cs

```diff
@@ -297,12 +300,21 @@ private bool AddPrefabRegistration(NetworkPrefab networkPrefab)
                 return true;
             }
 
+            // Make sure we don't have several overrides targeting the same prefab. Apparently we don't support that... shame.
+            if (OverrideToNetworkPrefab.ContainsKey(target))
+            {
+                // If so, just return false and don't attempt to add another.
+                return false;
```

Comment:
Weren't we going to leave this check out?

Reply:
😞 
Yes... it is in the wrong location... need to write another test.

Reply:
Ok... check the update... that is where that check should be.